packages/store-sync/src/watchLogs.ts

```diff
@@ -42,7 +42,6 @@ export function watchLogs({ url, address, fromBlock }: WatchLogsInput): WatchLog
 
       client = await getWebSocketRpcClient(getUncachedUrl(url), {
         keepAlive: true,
-        reconnect: { attempts: 100, delay: 1_000 },
```

Comment:
Removing the aggressive retries here and just using the default reconnect setting, since the sync logic already recreates the stream if something goes wrong.